src/lib/containers/QueryWrapper.js

```diff
@@ -74,15 +74,23 @@ export default class QueryWrapper extends React.Component {
                     }
                 )
                 data.facets = filteredData
+                // have to write this so test pass
+                if (!filteredData || !filteredData[0] ) {
```

Comment:
I think this could be cleaner.  Create the state object, and if (filteredData && filteredData[0]) then add the facetCount key?